catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPreStoragePlugin.java

```diff
@@ -256,26 +256,31 @@ private BufferedImage render(ContentItem contentItem,
         if (contentItem != null) {
             InputStream inputStream = contentItem.getInputStream();
 
-            if (inputStream != null) {
-                try {
-                    NitfRenderer renderer = new NitfRenderer();
-
-                    new NitfParserInputFlow().inputStream(inputStream)
-                            .allData()
-                            .forEachImageSegment(segment -> {
-                                if (bufferedImage.get() == null) {
-                                    BufferedImage bi =
-                                            imageSegmentFunction.apply(new ImmutablePair<>(segment,
-                                                    renderer));
-                                    if (bi != null) {
-                                        bufferedImage.set(bi);
+            try {
+                if (inputStream != null) {
+                    try {
+                        NitfRenderer renderer = new NitfRenderer();
+
+                        new NitfParserInputFlow().inputStream(inputStream)
+                                .allData()
+                                .forEachImageSegment(segment -> {
+                                    if (bufferedImage.get() == null) {
+                                        BufferedImage bi = imageSegmentFunction.apply(new ImmutablePair<>(
+                                                segment,
+                                                renderer));
+                                        if (bi != null) {
+                                            bufferedImage.set(bi);
+                                        }
                                     }
-                                }
-                            })
-                            .end();
-                } finally {
-                    IOUtils.closeQuietly(inputStream);
+                                })
+                                .end();
+                    } finally {
+                        IOUtils.closeQuietly(inputStream);
+                    }
                 }
+            } catch (IllegalArgumentException e){
```

Comment:
@lessarderic recommended a nice pattern to me recently. I think we should catch `IllegalArgumentException | RuntimeException`. This has the advantages of more explicit readability and not accidentally swallowing checked exceptions in the future.